Source/Core/Common/FileUtil.cpp

```diff
@@ -755,8 +772,10 @@ std::string GetSysDirectory()
 	sysDir = GetBundleDirectory() + DIR_SEP + SYSDATA_DIR;
 #elif defined (_WIN32)
 	sysDir = GetExeDirectory() + DIR_SEP + SYSDATA_DIR;
+#elif defined (LINUX_LOCAL_DEV)
+	sysDir = GetExeDirectory() + DIR_SEP + "Sys";
```

Comment:
On linux SYSDATA_PATH is defined as #define SYSDATA_DIR DATA_DIR "sys" (in Source/Core/Common/CommonPaths.h:43) meaning that it will always be an absolute path if the DATA_DIR is defined.
Maybe the CMakeLists.txt could be modified so that DATA_DIR is unset when LINUX_LOCAL_DEV is set?